src/msgpack/buffer.d

```diff
@@ -140,7 +140,7 @@ struct RefBuffer
      *  value = the content to write.
      */
     @trusted
-    void putRef(in ubyte[] value)
+    void putRef(const scope ubyte[] value)
```

Comment:
You say it's `scope`, but doesn't it store a reference?

Reply:
Yes, I've applied in batch the renaming of `in ubyte[]` to `const scope[] ubyte` so it's possible that this one is not necessary but at least it continue compiling w/ or w/o -dip1000